config/config.go

```diff
@@ -1477,6 +1489,13 @@ func InitClient() error {
 		viper.Set("Client.MinimumDownloadSpeed", downloadLimit)
 	}
 
+	// Ensure our worker count is set
+	if viper.IsSet("Client.WorkerCount") {
+		viper.SetDefault("Client.WorkerCount", param.Client_WorkerCount.GetInt())
```

Comment:
I don't think this check really makes sense. If you check and find the viper variable is already set, then setting a default has no effect because it's already set and always overridden. It looks like this value is also already set in `config/resources/defaults.yaml`, so setting a viper default shouldn't have an effect anyway.